### DISCUSSION TITLE
Feature Request: Individual corner radii

### DISCUSSION THREAD

**Discussion by jannuary** [uxbox/uxbox#535] [Ideas]
Control over corner radius of individual corners, rather than the entire shape is a must-have. 

Figma, for instance, has this:
![image](https://user-images.githubusercontent.com/27908024/106705295-435bac00-6620-11eb-9ebb-1950e358a425.png)

**Comment by myfunnyandy** [comment]
Hi, editing individual corners for border radius is already prioritized in our roadmap, it shouldn't be long until you have it in Penpot :)
